src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/integration/pact/SetupAssistant.kt

```diff
@@ -144,7 +144,7 @@ class SetupAssistant(
   fun createEndedReferral(id: UUID = UUID.randomUUID(), intervention: Intervention = createIntervention(), cancellationReason: CancellationReason? = randomCancellationReason(), cancellationComments: String? = null): Referral {
```

Comment:
We should probably change cancellationReason/comments etc in the method parameters to the new names.

Reply:
done